src/views/Swap/components/SwapModalHeader.tsx

```diff
@@ -40,14 +45,8 @@ export default function SwapModalHeader({
 
   const tradeInfoText =
     trade.tradeType === TradeType.EXACT_INPUT
-      ? t('Output is estimated. You will receive at least %amount% %symbol% or the transaction will revert.', {
-          amount,
-          symbol,
-        })
-      : t('Input is estimated. You will sell at most %amount% %symbol% or the transaction will revert.', {
-          amount,
-          symbol,
-        })
+      ? t('Output is estimated. You will receive at least or the transaction will revert.')
```

Comment:
why removing `%amount% %symbol%` here?

Reply:
Designer want to simplify the sentence. I think